### PR TITLE
Hackerthon Readme Repo update

### PR DESCRIPTION
**Pull request by cryptoade1**
- Mention that we provide two tools (request and verify)

- Step-by-step (more detailed) guide on how to request/verify VC, ideally with screenshots

- Explanation of key components (e.g. attesting ETH address, how is the proofValue calculated, etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,18 +84,64 @@ npm run dev
...
+- Populate/self-create the VC content, but it should be **on-chain  and/or web2** identity-related information
+- Find the VC use case in your dApp (what are you going to do with the VCs?)
+- Extend the web service to diversify the ways to integrate the VC into your dApps.
+
+Example: instead of returning the VC as bytes to the user, you can:
+    - upload the VC bytes as SBT
+    - trigger smart contract using the VC content
+    - integrate the VC into wallets and manage/use VCs from there
+
+**TODO:**
```
Comment (Kailai-Wang) [inline README.md:133]: I have you have to remove this `TODO` line and make the `Must haves` and `Nice-to-haves` in bold text

File: README.md [hunk 2]
```diff
@@ -12,13 +12,29 @@ Before starting the development server, please follow these steps:
...
+
+![image (7)](https://github.com/cryptoade1/vc-oracle-demo/assets/88367184/30a2cddb-7401-42d0-8a6b-db297ca777a0)
+
+Then install [Hardhat](https://hardhat.org/) and have it set up in your project.
+
+![image (8)](https://github.com/cryptoade1/vc-oracle-demo/assets/88367184/a761da45-b44a-4ed7-aef1-ff693269c6c4)
+
+Then run the command to call hardhat;
+
+![image (9)](https://github.com/cryptoade1/vc-oracle-demo/assets/88367184/c98d8ec3-261f-4d01-997a-837b72d2c358)
```
Comment (Kailai-Wang) [inline README.md:35]: Maybe we don't need to add screenshots for hardhat installation as it's basic for hackers :)

**Review by Kailai-Wang — APPROVED** [review]
Looks good in general, thanks!

**Comment by YoshiyukiSakura** [comment]
lgtm

**Comment by YoshiyukiSakura** [comment]
refers P-235

File: README.md (hunk 1 above)
Comment (cryptoade1) [reply README.md:133]: Done!

File: README.md (hunk 2 above)
Comment (cryptoade1) [reply README.md:35]: Haha, I'll prefer it :)

**Pull request merged**